go/contacts/contacts.go

```diff
@@ -35,96 +29,83 @@ func ResolveContacts(mctx libkb.MetaContext, provider ContactsProvider, contacts
 		return res, nil
 	}
 
-	// Collect phone numbers and emails from Contact list. Iterate through
```

Comment:
This function (`ResolveContacts`) is almost entirely new - it's best to just go to that file and look rather than looking at the diff.